extensions/amp-story-shopping/0.1/amp-story-shopping.md

```diff
@@ -23,7 +23,7 @@ scripts:
 
 # amp-story-shopping
 
-<amp-img alt="An example of amp story shopping tag in each type of configuration" src="https://user-images.githubusercontent.com/3860311/152416815-61a63a88-76c0-4ae8-9300-a7ad277dbe7a.jpg" layout="intrinsic" width="806" height="428">
+<amp-img alt="An example of amp story shopping tag in each type of configuration" src="https://user-images.githubusercontent.com/3860311/152416815-61a63a88-76c0-4ae8-9300-a7ad277dbe7a.jpg" layout="intrinsic" width="806" height="428"></amp-img>
```

Comment:
This one has the  same description as the one below it,
but no dashes "`-`" 
`amp story shopping tag` vs. `amp-story-shopping-tag`

If the alt attribute description is correct, you will need to make the dashes consistent

However, the description for the alt attribute doesn't seem to match the heading, it seems to be the same for the shopping tag, double check that this is the correct description for this section.